sklearn/datasets/_openml.py

```diff
@@ -734,6 +741,21 @@ def _valid_data_column_names(features_list, target_columns):
     return valid_data_column_names
 
 
+@validate_params(
+    {
+        "name": [str, None],
+        "version": [Interval(Integral, 1, None, closed="left"), StrOptions({"active"})],
+        "data_id": [Interval(Integral, 1, None, closed="left"), None],
+        "data_home": [str, None],
+        "target_column": [str, list, None],
+        "cache": [bool],
+        "return_X_y": [bool],
+        "as_frame": [bool, StrOptions({"auto"})],
+        "n_retries": [Interval(Integral, 1, None, closed="left")],
+        "delay": [Interval(Real, 0, None, closed="right")],
+        "parser": [StrOptions({"auto", "pandas", "liac-arff", "warn"})],
```

Comment:
The [CI failure](https://github.com/scikit-learn/scikit-learn/pull/26519) looks real. The `read_csv_kwargs` parameter needs to be specified.